packages/datadog-instrumentations/src/pg.js

```diff
@@ -139,35 +136,28 @@ function wrapQuery (query) {
 
       try {
         return retval
-      } catch (err) {
-        errorCh.publish(err)
+      } catch (error) {
+        ctx.error = error
+        errorCh.publish(ctx)
       }
     })
   }
 }
-
+const finish = () => {
+  finishPoolQueryCh.publish()
```

Comment:
This needs to receive the context, otherwise it may not be able to figure out the current store.

Reply:
Done 😃